app_dev.yaml

```diff
@@ -262,6 +262,8 @@ env_variables:
 # redirects to these instructions to disable URL fetch:
 # https://cloud.google.com/appengine/docs/standard/python/sockets#making_httplib_use_sockets
   GAE_USE_SOCKETS_HTTPLIB : "anyvalue"
+  # NOTE TO RELEASE COORDINATORS: Point this to the appropriate path before deploying the server.
+  GOOGLE_APPLICATION_CREDENTIALS: "/path/to/service-account-file.json"
```

Comment:
If this is for the Admin SDK, I don't think we should have it in the source code. Please let me know if it would be helpful to discuss this further.